packages/concerto-core/lib/serializer/jsongenerator.js

```diff
@@ -42,15 +42,13 @@ class JSONGenerator {
      * written for subsequent instances, false by default.
      * @param {boolean} [convertResourcesToId] Convert resources that
      * are specified for relationship fields into their id, false by default.
-     * @param {boolean} [ergo] target ergo.
      * @param {number} [utcOffset] UTC Offset for DateTime values.
      */
-    constructor(convertResourcesToRelationships, permitResourcesForRelationships, deduplicateResources, convertResourcesToId, ergo, utcOffset) {
+    constructor(convertResourcesToRelationships, permitResourcesForRelationships, deduplicateResources, convertResourcesToId, utcOffset) {
```

Comment:
Looks like we also need a dummy param here?

Reply:
Oops! missed that one.